cmd/siac/rentercmd.go

```diff
@@ -470,11 +470,14 @@ func renterfilesdeletecmd(path string) {
 // Downloads a path from the Sia network to the local specified destination.
 func renterfilesdownloadcmd(path, destination string) {
 	destination = abs(destination)
+	var err error
 	done := make(chan struct{})
-	go downloadprogress(done, path)
-
-	err := httpClient.RenterDownloadFullGet(path, destination, false)
+	if !renterDownloadAsync {
+		go downloadprogress(done, path)
+	}
+	err = httpClient.RenterDownloadFullGet(path, destination, renterDownloadAsync)
 	close(done)
+
```

Comment:
Can you remove this newline? We want to make sure the line in which the `err` is assigned is close to the `if` that checks the error.